.github/workflows/simorgh-local-server-tests.yml

```diff
@@ -35,24 +35,25 @@ jobs:
 
       - name: Install Node Modules
         if: steps.cache.outputs.cache-hit != 'true'
-        run: npm ci
+        run: yarn install --frozen-lockfile
 
       - name: Build Simorgh
-        run: npm run build:local
+        run: yarn run build:local
 
       - name: Start Simorgh Server
         run: nohup node build/server.js > /dev/null 2>&1 &
 
       - name: Run AMP Validator
-        run: npm run amp:validate
+        run: yarn run amp:validate
 
       - name: Run bbc-a11y
-        run: npm run bbcA11y:ci
+        run: yarn run bbcA11y:ci
 
       - name: Run Lighthouse
-        run: npm run lighthouse
+        run: yarn run lighthouse
 
       - name: Run Puppeteer Tests
         run: |
-          npm install --no-save puppeteer
-          npm run test:puppeteer
+          yarn add puppeteer
```

Comment:
We might be able to split this into installing puppeteer and running the tests. That way we can skip the puppeteer installation if it's in the cache.